Clarify multi-line string text (#5140)
dart-lang/site-www: src/language/built-in-types.md

assert(s2 == 'The + operator works, as well.');
```

> You create a multi-line string by using a triple quote with

Reviewer: ```suggestion
To create a multi-line string, use a triple quote with
```
